README.rst

```diff
@@ -58,12 +58,164 @@ Python:
   provides an abstraction of NI-DAQmx in the ``nidaqmx`` module, which collides
   with this package's module name.
 
+Getting Started
+===============
+
+In order to use the **nidaqmx** package, you must have at least one DAQ device installed on your
+machine. X series DAQ device (eg: PCIe-6363 or USB-6351) is recommended, physical or
```

Comment:
"X Series" is a proper noun.

"DAQ device" needs an article.

Use "e.g." not "eg:".

Don't use run-on sentences.

"Physical or simulated devices": it's more appropriate to use "and", because both classes of devices are supported. If you were describing a single device, like "A physical or simulated device is supported", then "or" would be more appropriate.

```suggestion
machine. An X Series DAQ device (e.g. PCIe-6363 or USB-6351) is recommended. Physical and
```